.changeset/fifty-berries-learn.md

```diff
@@ -0,0 +1,40 @@
+---
+'@backstage/plugin-catalog': patch
+---
+
+---
+
+This change would allow consumers of plugin catalog to make use of labels just like tags from metadata.
```

Comment:
This will show up in the changelog so it's better to phrase it more around that this feature is added and how it works, if someone wants to know the reasoning they typically find that in the PR description.

A start might be.

> Added new `labelColumn` to `CatalogTable.column` this new column allows you to...

Reply:
Rephrased the changeset with suggested changes in latest commit.